src/commands/run/jobs-builder/deploy-job-builder.ts

```diff
@@ -19,6 +19,10 @@ interface DeployJobMobileWikiParams {
   crowdin_branch: string;
 }
 
+interface DeployPlatformsParams {
+  BRANCH: string;
```

Comment:
Why is it all caps?

Reply:
The jenkins job expecting it to be uppercase. Lowercase `branch` didn't worked